samples/nrf9160/modem_shell/src/th/th_shell.c

```diff
@@ -34,7 +34,17 @@ static int cmd_th_startfg(const struct shell *shell, size_t argc, char **argv)
 		mosh_error("please, give also the command to be run");
 		return -EINVAL;
 	}
-	th_ctrl_start(shell, argc, argv, false);
+	th_ctrl_start(shell, argc, argv, false, false);
+	return 0;
+}
+
+static int cmd_th_start_pipeline(const struct shell *shell, size_t argc, char **argv)
+{
+	if (argc < 2) {
+		mosh_error("please, give also the command to be run");
```

Comment:
We don't need to be polite here :)

```suggestion
		mosh_error("Missing the command to be run");
```